src/datachain/lib/audio.py

```diff
@@ -160,39 +163,82 @@ def audio_fragment_bytes(
     return buffer.getvalue()
 
 
-def save_audio_fragment(
+def save_audio(
```

Comment:
I just wonder if we need to just save audio (fragment) to a file, will it be faster and easier to use, let's say, `ffmpeg` rather than to use audio_to_np and torchaudio 🤔